test-failure-policy

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env python3
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2021 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import argparse
+import datetime
+import fnmatch
+import re
+import os
+import socket
+import sys
+import traceback
+
+sys.dont_write_bytecode = True
+
+from task import github
+from lib.constants import BOTS_DIR
+from lib.testmap import get_test_image
+
+
+def main():
+    script = os.path.basename(__file__)
+    parser = argparse.ArgumentParser(description='Check a traceback for a known issue')
+    parser.add_argument('-o', "--offline", action='store_true',
+                        help="Work offline, don't fetch new data or contact servers")
+    parser.add_argument('image', help="The image to check against")
```

Comment:
Was it a conscious decision to replace context with image? I wonder if that makes the caller's job harder -- right now you can just use `$TEST_OS`, and tests-policy would strip off the scenario.

Reply:
It was. 

- It was really only interested in image
- Tests only pass image name, never scenario
- The parser in `tests-policy` is not correct (it works because it is called only with image name)
- Tests-policy assumes that default contex looks like `verify/<OS>` which hasn't been true for a while now

So I thought that rather than doing the whole parsing of context etc. just so we can call it manually a bit differently felt a bit pointless so I simplified it as much as possible. You can just do `./test-failure-policy fedora-33` instead of `./test-failure-policy fedora-33/firefox`

Reply:
If the tests only pass the image name, and that's not much harder than passing `$TEST_OS`, then agreed. Thanks!